site/content/3.12/aql/functions/geo.md

```diff
@@ -7,6 +7,383 @@ description: >-
+interior of the MultiPolygon if and only if it is
+contained in one of the polygons. If some polygon P2 in a MultiPolygon
+is contained in another polygon P1, then P2 is treated like a hole
+in P1 and containment of points is defined with the even-odd-crossings rule
+(see [Polygon](#polygon)).
+
+Additionally, the following rules apply and are enforced for
+MultiPolygons:
+
+- No two edges in the linear rings of the polygons of a MultiPolygon
+  may intersect.
+- Polygons in the same MultiPolygon may not share edges, but they may share
+  coordinate pairs.
+
+Example with two polygons, the second one with a hole:
+
+```json
+{
+    "type": "MultiPolygon",
+    "coordinates": [
+        [
+            [
+                [102.0, 2.0],
+                [103.0, 2.0],
+                [103.0, 3.0],
+                [102.0, 3.0],
+                [102.0, 2.0]
+            ]
+        ],
+        [
+            [
+                [100.0, 0.0],
+                [101.0, 0.0],
+                [101.0, 1.0],
+                [100.0, 1.0],
+                [100.0, 0.0]
+            ],
+            [
+                [100.2, 0.2],
+                [100.2, 0.8],
+                [100.8, 0.8],
+                [100.8, 0.2],
+                [100.2, 0.2]
+            ]
+        ]
+    ]
+}
+```
+
+## GeoJSON interpretation
+
+Note the following technical detail about GeoJSON: The
+[GeoJSON standard, Section 3.1.1 Position](https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.1)
+prescribes that lines are **cartesian lines in cylindrical coordinates**
+(longitude/latitude). However, this definition is inconvenient in practice,
+since such lines are not geodesic on the surface of the Earth.
+Furthermore, the best available algorithms for geospatial computations on Earth
+typically use geodesic lines as the boundaries of polygons on Earth.
+
+Therefore, ArangoDB uses the **syntax of the GeoJSON** standard,
+but then interprets lines (and boundaries of polygons) as
+**geodesic lines (pieces of great circles) on Earth**. This is a
+violation of the GeoJSON standard, but serving a practical purpose.
+
+Note in particular that this can sometimes lead to unexpected results.
+Consider the following polygon (remember that GeoJSON has
+**longitude before latitude** in coordinate pairs):
+
+```json
+{ "type": "Polygon", "coordinates": [[
+  [4, 54], [4, 47], [16, 47], [16, 54], [4, 54]
+]] }
+```
+
+![GeoJSON Polygon Geodesic](../../../images/geojson-polygon-geodesic.webp)
+
+It does not contain the point `[10, 47]` since the shortest path (geodesic)
+from `[4, 47]` to `[16, 47]` lies North relative to the parallel of latitude at
+47 degrees. On the contrary, the polygon does contain the point `[10, 54]` as it
+lies South of the parallel of latitude at 54 degrees.
+
+{{< info >}}
+ArangoDB version before 3.10 did an inconsistent special detection of "rectangle"
+polygons that later versions from 3.10 onward no longer do, see
+[Legacy Polygons](#legacy-polygons).
```

Comment:
The Legacy Polygons section is still in the Index page. Perhaps you wanted to move it here as well? Otherwise, the link needs to be updated.

Reply:
No, it needs to stay where it is

> For backward compatibility, a `legacyPolygons` option has been introduced
for geo indexes.